CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKRenderer.java

```diff
@@ -686,6 +699,19 @@ public void Display() {
         cvkVisualProcessor.signalProcessorIdle();
     }    
     
+   /**
+     * @param file to save the screenshot to
+     */
+    public CVKRenderUpdateTask TaskRequestScreenshot(File file) {
+        //=== EXECUTED BY CALLING THREAD (VisualProcessor) ===//
+        requestScreenshot = true;
```

Comment:
Does it work if you move these assignments to lambda?

Reply:
Yes it does

Reply:
It was moved